src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs

```diff
@@ -49,5 +59,21 @@ public override string ToString()
         {
             return $"Key: {Key}, Path: {Path}, VR:{VR}, Level:{Level}, Status:{Status}";
         }
+
+        public override int GetHashCode()
+        {
+            return HashCode.Combine(Key.GetHashCode(), Path, VR, Level.GetHashCode(), Status.GetHashCode());
```

Comment:
VR could be null, would HashCode.Combine works with that?

Reply:
I believe so! 

---
In reply to: [558518647](https://github.com/microsoft/dicom-server/pull/501#discussion_r558518647) [](ancestors = 558518647)

Reply:
Yeah: [https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/HashCode.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/HashCode.cs)

---
In reply to: [558589887](https://github.com/microsoft/dicom-server/pull/501#discussion_r558589887) [](ancestors = 558589887,558518647)